curriculum/js1/5.js

```diff
@@ -5,8 +5,10 @@
  * @param {function} b
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (num, funct) => {
+  if (num === 0) { return };
```

Comment:
`if (num === 0) { return; }` is correct because we dont end the code block after a conditional with `;`

But if you want to code to Garagescript coding conventions it is `if (num === 0) return;`

Reply:
fixed!